kubernetes/ccloudexporter.crd.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    release: monitoring
+    app: ccloudexporter
+  name: ccloudexporter
+spec:
+  jobLabel: ccloudexporter
+  endpoints:
+  - path: /metrics
+    port: metrics
+    interval: 30s
```

Comment:
Maybe we should increase this interval (to 60s). The reason behind this is that each call to `/metrics` generates multiple queries to the Metrics API (and they are quite expensive).
Note for myself: do a `/health` checkpoint ;)

Reply:
ok, you are the expert here ;)